sdk/table/tests_invoke.yml

```diff
@@ -0,0 +1,42 @@
+parameters:
```

Comment:
wondering if we need to add this file to tables. I think all of storage shares it.